docs/src/main/paradox/server/details.md

```diff
@@ -44,43 +44,14 @@ Java
 :   @@snip[ExceptionGreeterServiceImpl](/interop-tests/src/test/java/example/myapp/helloworld/grpc/ExceptionGreeterServiceImpl.java) { #streaming }
 
 ## Rich error model
-Beyond status codes you can also use the [Rich error model](https://www.grpc.io/docs/guides/error/#richer-error-model). Currently there is no native support for this concept in Akka gRPC. However you can use the following manual approach.
-
-Add the following dependency to receive required classes (that are based on the [common protobuf](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto)):
```

Comment:
I think it was useful to include a reference to https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto in the docs - sure, it's linked from https://www.grpc.io/docs/guides/error/#richer-error-model as well but I've found myself searching for it in the past ;)

Reply:
Yes we can keep it, especially since it is used in the spec